samples/snippets/transcribe_async_gcs.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
Update date for net-new samples.

Reply:
Just double-checking; here you mean updating 2017->2021, correct?  I just made this change in the most recent commit.